frontend/src/components/charts/BatteryLikeChart/index.tsx

```diff
@@ -0,0 +1,53 @@
+import "./styles.scss";
```

Comment:
NIT: relative imports should be at the bottom

Reply:
Oh, nice catch! I always try to review my imports to make them in this order, but I missed this one. Maybe we can add it to the lint rules later, what do you think? Anyway, it's solved now

Reply:
I think we should have this lint rule already but maybe it's not working properly because of the folder structure. :thinking:

Reply:
I will take a look then